lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -20,18 +20,21 @@
     % elif not credit_status["purchased"]:
         <p class="credit-eligibility-msg">
             % if credit_status["deadline"] < datetime.datetime.now(pytz.UTC) + datetime.timedelta(days=30):
-                ${_("The opportunity to purchase credit for this course expires on {deadline}. You've worked hard - don't miss out!").format(deadline=get_default_time_display(credit_status["deadline"]))}
```

Comment:
The current version of the messages in this template have gone through doc review, so please change these back to the version on master.